### DISCUSSION TITLE
An issue occurs when displaying a project on a website.

### DISCUSSION THREAD

**Discussion by S3VENK** [subquery/subql#1020] [Frontier TestNet - Bug Reporting]
When I visited the website  http://IP:8000, I saw various errors.
<img width="855" alt="bug" src="https://user-images.githubusercontent.com/94055429/167915550-9b7d2da2-4be0-4b70-ace0-eae55347f91a.png">
....................................................................................................................................................

<img width="820" alt="non bug" src="https://user-images.githubusercontent.com/94055429/167915561-886f45d9-cb6f-4f6d-81a8-53766b7ad0b9.png">

#bug

**Comment by mzxyz** [comment]
Resolved in `coordinator:v0.1.10`
